src/main/autoupdater.js

```diff
@@ -1,20 +1,12 @@
 'use strict'
-
-const { app, autoUpdater, dialog } = require('electron')
-const os = require('os')
-const updaterFeedURL = 'https://selfkey-releases.storage.googleapis.com/SelfKey-Desktop-Identity-Wallet/ea788838ccc9eb63676dd2da87772850/' + os.platform() + '/' + os.arch() + '/RELEASES.json?method=JSON&version=' + app.getVersion()
```

Comment:
check with @dfcowell if we are still gonna use nucleus server

Reply:
@altninja good point. I have setup auto update with github release. I guess we can point a custom domain to it from the selfkey website. Something like this https://help.github.com/articles/linking-to-releases/ 

@dfcowell thoughts?

Cheers

Reply:
Let's keep using Github releases. Less to maintain. :+1: